requirements.txt

```
@@ -1,2 +1,2 @@
-tensorflow>=2.3,<2.5
+tensorflow>=2.3
```

```suggestion
tensorflow>=2.3,<2.6
```
TF is prone to break with version upgrades, so better be safe